source/important_details/includes.md

```diff
@@ -29,23 +29,24 @@ appending the `require_once` at the end of your configuration file.
 
 Drupal-specific configuration snippets are stored in `../config/drupal`. 
 
-Example: `settings.php` for a Drupal 7 site
+Example: `settings.php` for a Drupal 8 site
 
 ```php
 <?php
 ...
-require_once('../config/drupal/settings-d7-site.php');
-require_once('../config/drupal/settings-d7-db123.php');
+require_once('../config/drupal/settings-d8-site.php');
+require_once('../config/drupal/settings-d8-db123.php');
 ?>
 ```
 
 The following configuration files are available:
 
-* `settings-d7-site.php` -- General website settings, for example Reverse Proxy
+* `settings-d8-site.php` -- General website settings, for example Reverse Proxy
   settings.
-* `settings-d7-dbXXX.php` -- Database configuration. Replace "dbXXX" with the
+* `settings-d8-dbXXX.php` -- Database configuration. Replace "dbXXX" with the
   database name of your site instance. 
-* `settings-d7-memcache.php` -- Memcache configuration. See "[How do I enable memcached?](/tech_faq/memcache.html)".
+* `settings-d8-memcache.php` -- Memcache configuration.
```

Comment:
remove this since the module is not ready yet.
See https://www.drupal.org/project/issues/memcache?status=Open&priorities=All&categories=All&version=8.x-2.x-dev&component=All